### PR TITLE
Add outputs for Cloud Run Jobs

### PR DESCRIPTION
**Pull request by lock14**
* Add outputs for Cloud Run Job service accounts
* Removed broken Cloud Run Job deployment entries in `.github/workflows/ci.yml`. I will add them back once the  service accounts are configured correctly

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -185,62 +185,6 @@ jobs:
             --image="${{ env.DOCKER_REPO }}/github-metrics-aggregator:${{ env.DOCKER_TAG }}-amd64" \
             --tag="${{ env.TAG_ID }}"
 
-  deployment_artifacs_job_integration:
```
Comment (verbanicm) [inline .github/workflows/ci.yml:188]: Why were these removed?

Comment (verbanicm) [reply .github/workflows/ci.yml:188]: nvm didnt read body went straight to code

**Review by verbanicm — APPROVED** [review]

**Review by sethvargo — APPROVED** [review]
